lib-integration-java/src/main/resources/acord/TL_103.xml

```diff
@@ -0,0 +1,171 @@
+<TXLife xmlns="http://ACORD.org/Standards/Life/2">
```

Comment:
Could we add these files to the `example-acord-app` directory instead? Actually... please remove these files.